### PR TITLE
Add comments

### PR DESCRIPTION
**Pull request by Rockev1n**

### CODE REVIEW

**Comment by amilcarlucas** [comment]
This needs a `git rebase master` and non-english will make the code harder to read.

File: ArduPlane/quadplane.cpp [hunk 1]
```diff
@@ -1250,8 +1250,8 @@ void QuadPlane::get_pilot_desired_lean_angles(float &roll_out_cd, float &pitch_o
         pitch_out_cd *= ratio;
     }
 
-    // apply lateral tilt to euler roll conversion
-    roll_out_cd = 100 * degrees(atanf(cosf(radians(pitch_out_cd*0.01))*tanf(radians(roll_out_cd*0.01))));
+    // apply lateral tilt to euler roll conversion   
+    roll_out_cd = 100 * degrees(atanf(cosf(radians(pitch_out_cd*0.01))*tanf(radians(roll_out_cd*0.01))));//不知道他在说啥
```
Comment (nkruzan) [inline ArduPlane/quadplane.cpp:1254]: Translation: I don't know what he's talking about

Not really appropriate comment for the code.

**Review by nkruzan — COMMENTED** [review]
I don't feel that comments that are translating other comments are really needed. Some of the comments may have value helping to understand what the code is doing if they were in English, but others are explaining C++ library functions or attempting to explain math. A couple are just noise.

**Review by khancyr — CHANGES_REQUESTED** [review]
This looks like only some comments in an asian type language (sorry I am unable to guess which one).
We only accept english contribution.
For a correct submission, I invite you to read https://ardupilot.org/dev/docs/contributing.html

**Comment by khancyr** [comment]
I am closing this for now. Feel free to reopen another PR with only your comments in english.

**Pull request closed without merging**